### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by fickryiman**
- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for reusable components: /components.
- Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup the router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App>
- Add 2 routes in your application, each should render a page component:
   /
   /categories
- Create components inside of the /components directory:
   Contains individual book state, as well as a button to delete the book
   Contains a list to renders individual books
   Contains a form to create new books
- Contains a navigation to link to the 2 routes created
- input for a category - replace it with an author

### CODE REVIEW

File: src/components/Input.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    category: '',
+  });
+
+  // const authors = ['Fickry Bil Iman', 'Suzanne Collins', 'Frank Herbert'];
+
+  // const authorOptions = authors.map((author) => (
+  //   <option value={author} key={author.id}>
+  //     {author}
+  //   </option>
+  // ));
```
Comment (thecodechaser) [inline src/components/Input.jsx]: - [x] Please make sure to remove the commented code here. It's not a good practice to leave commented code on the codebase.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @fickryiman👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Input.jsx (hunk 1 above)
Comment (fickryiman) [reply src/components/Input.jsx]: removed commented code

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @fickryiman, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**